### ISSUE TITLE
Release 2025.1

### ISSUE THREAD

**Issue by pkalita-lbl** [microbiomedata/issues#988]
Release 2025.1 is scheduled to be deployed to production on: **February 3, 2025**.

### Events
* **January 13, 2025** (Monday): ✏️ Release prep meeting
*  ~~January 15, 2025~~ **January 17, 2025** (Friday): 🧊 Freeze the schema that will be introduced in this release
* ~~January 20, 2025~~ **January 21, 2025** (Tuesday): 🧪 Begin testing final software versions in dev environments (moved to avoid MLK Day holiday)
* ~~January 27, 2025~~ **February 3, 2025** (Monday): 🚀 Deploy to production (moved to avoid NERSC planned power outage)

### Non-standard tasks/steps
If there is a task/step you want the release process to include, which isn't already part of the [standard release process](https://github.com/microbiomedata/infra-admin/tree/main/releases) (documented in the private infra-admin repo), please describe it in a comment below.

### Blocking issues
-

**Comment by eecavanna** [comment]
Non-standard step for this release:
- [ ] Login to Dagit and enable the `daily_ensure_alldocs` thing on the "Automation" page (see [this PR comment](https://github.com/microbiomedata/nmdc-runtime/pull/810#issuecomment-2549652071) for a related screenshot from the development environment).

**Comment by eecavanna** [comment]
Non-standard step for this release—to take place at any time (since the Runtime does not use this Mongo collection):
- [ ] Connect to the Mongo server as a user that has sufficient privileges to drop the `metap_gene_function_aggregation` collection (e.g. the `root` user). Then, drop that collection.

**Comment by eecavanna** [comment]
Non-standard step for this release (only being included as part of "Release Day" because it involves a brief period of downtime):
- [ ] Update the `mongo` deployment from 6.0.4 to 7.0.15
  - Procedure: Scale down all Mongo-dependent workloads, run `backup-mongo` to create a pre-upgrade dump (just in case), scale down `mongo` workload, edit its configuration so the container image is `bitnami/mongodb:7.0.15`, scale it back up, perform sanity test query using Mongo a client (e.g. `db.getCollection("study_set").find({});`), scale up all Mongo-dependent workloads.
- [ ] Update the `backup-mongo` CronJob from 6.0.4 to 7.0.15
  - Procedure: Edit its configuration so the container image is `bitnami/mongodb:7.0.15`. That's all.

**Comment by eecavanna** [comment]
Non-standard step for the release: **Removing banners**
- [ ] Remove the downtime banner from the Data Portal (environment variable on Spin), in dev and prod
- [ ] Remove the downtime banner from the Runtime (environment variable on Spin), in dev and prod
- [ ] Remove the status message from https://status.microbiomedata.org/ (i.e. delete the contents of https://github.com/microbiomedata/nmdc-status-website/blob/main/message.md)

**Comment by eecavanna** [comment]
FYI: I can do (i.e. take care of) all of the non-standard steps listed above. I'll say this in the upcoming release meeting also.

**Comment by pkalita-lbl** [comment]
Done.

Next: https://github.com/microbiomedata/issues/issues/1046
